headers/types/common/kaomado.h

```diff
@@ -0,0 +1,54 @@
+// Represents the state of a portrait that wants to be displayed
+struct kaomado {

```

Comment:
Could we either write a "kaomado 101" like we did with the WTE format, or just link to the exiting Project Pokémon doc? It'll help someone looking at this without prior knowledge.

Reply:
Oh yeah! Personally I'd prefer to write it, as there's a few aspects that aren't covered/mentioned there (how portraits are looked up, the role of negative pointers as size boundaries, how missing emote look-ups are resolved...) Although where would it go exactly?

Reply:
I would probably put an overview of the format at the top of this file, or in the symbol description of one of the key functions. I would put nitty-gritty details related to a specific function or struct in the relevant place (the description of that function symbol, or as a comment over the relevant struct). The organization of the memory allocation stuff might be a good example (see `struct mem_*` in `common.h` and the `MemAlloc` family in `arm9.yml`).